src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,132 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String sqlRequest = "INSERT INTO manufacturers "
+                + "(name, country) VALUES (?, ?);";
+        try (
+                Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement manufacturerCreateStatement =
+                        connection.prepareStatement(sqlRequest, Statement.RETURN_GENERATED_KEYS)
+        ) {
+            manufacturerCreateStatement
+                    .setString(1, manufacturer.getName());
+            manufacturerCreateStatement
+                    .setString(2, manufacturer.getCountry());
+            manufacturerCreateStatement.executeUpdate();
+            ResultSet resultSetUpdate = manufacturerCreateStatement.getGeneratedKeys();
+            resultSetUpdate.next();
+            Long id = resultSetUpdate.getObject(1, Long.class);
+            manufacturer.setId(id);
```

Comment:
```suggestion
            if (resultSetUpdate.next()) {
            	Long id = resultSetUpdate.getObject(1, Long.class);
            	manufacturer.setId(id);
            }
```

Why did you remove the if statement? What does `resultSetUpdate.next()` method do? (Same un other methods)